spec/models/like_spec.rb

```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
```

Comment:
Please ensure that you add unit tests for the `Comment`, and `Like` models according to the project requirement.